winter-website.html

```diff
@@ -12,6 +12,15 @@
 
 <body>
 	
+	<div class="navBar">
+		<div class="navBarLinks">
+			<a class="navBarText" href="#">NavBar</a>
```

Comment:
in consistent class naming here. think it should be navLinksText not navBarText in first a

Reply:
you sorted this already =]